### DISCUSSION TITLE
Incorrect Data Frame creation

### DISCUSSION THREAD

**Discussion by akshitarora4259** [nightscape/spark-excel#775] [Q&A]
Hi,

I am working with an Excel file, which contains data in the following format:
1. The first column of the first row contains a Note specifying the details of the data, the file contains.
2. The second row is empty.
3. The third row onwards, contains the actual data which has 5 columns of data.

When I am creating a data frame using this Excel file using the following code snippet:
df = spark.read.format("com.crealytics.spark.
[incorrect dataframe excel.xlsx](https://github.com/crealytics/spark-excel/files/12548516/incorrect.dataframe.excel.xlsx)
excel").option("header", False).option("dataAddress", "'Sheet3'!").load("file.xlsx")

Though, my data contains 5 columns, in the final data frame I can see only 1 column.

Can you please help me resolve this issue, so I can get the expected data frame containing all the columns of the data.

I am attaching the same of the excel file for reference.

**Comment by nightscape** [comment]
Your `dataAddress` looks broken. It should contain a cell range, so rather sth. like `"'Sheet3'!A1:E30"`

**Reply by akshitarora4259** [reply]
I tried using the cell range as well. Still it's not producing the required result. Also, our data is dynamic, so won't be able to pass the cell range for every scenario.
